### DISCUSSION TITLE
Problem with MetData processor

### DISCUSSION THREAD

**Discussion by AlexKDo** [UMEP-dev/UMEP#513] [Q&A]
Hello!
I am trying to upload my meteo daa and i get this error:
![image](https://user-images.githubusercontent.com/81244161/228697515-a71d7708-984b-4f77-b8cb-604c0a851e84.png)

file i want to upload is in CSV UTF- 8Format

**Comment by biglimp** [comment]
Can you paste in the first 10 lines of your csv-file.

**Comment by gusbacos** [comment]
It looks like your .csv file has ; as Column Separator. Change to Semicolon (;) and try load the data again.
